commands/clearroles.js

```diff
@@ -0,0 +1,40 @@
+const Discord = require("discord.js")
+const reaction_message_model = require("../models/reaction_roles_message.js")
+const mongoose = require('mongoose');
+const conf = require('../conf.js');
+mongoose.connect(conf.mongodbs, { useNewUrlParser: true })
+
+/**
+ * Clears all the roles on any message in the server
+ */
+module.exports = async (bot, message, args) => {
+    if (args[0] == "help"){
+        let help = new Discord.RichEmbed()
+        .setColor("#FFFF00")
+        .addField("Usage", "clearroles")
+        .addField("More info", "Clears all the roles on any message in the server.")
+        return message.channel.send(help).then(r => r.delete(30000))
+    }
+    if(!message.member.hasPermission("MANAGE_ROLES")){
+        message.reply("You don't have permission").then(async m=>{
+            await message.delete()
+            return m.delete(15000)
+        })
+    }
+    reaction_message_model.deleteMany({
+        guildID: message.guild.id
+    },(err, rem) => {
```

Comment:
Unless necessary (i.e. to avoid rebinding of this) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.
```suggestion
    }, function(err, rem) {
```